### DISCUSSION TITLE
Reindex from Settings same as from Wallet menu?

### DISCUSSION THREAD

**Discussion by nico89-exp** [DeFiCh/app#813] [Q&A] [answered]
Hi @izzycsy,

Is there a difference between the two available reindex options?

One is in the settings, one in the wallet menu. Is one of those faster (perhaps)? Is the wallet reindex bringing back "lost" transactions (cf. our other discussion)?

Best,
Mitja

![screen1](https://user-images.githubusercontent.com/80567701/117357169-e5579280-aeb4-11eb-9332-9b185476efb0.png)
![screen2](https://user-images.githubusercontent.com/80567701/117357175-e688bf80-aeb4-11eb-9d41-f3c63c192a9d.png)

**Comment by izzycsy** [comment, marked as answer]
Hi @nico89-exp , there's no differences and serves the same purpose, just another user-flow for users to access.
